acmwebsite/controllers/project.py

```diff
@@ -0,0 +1,14 @@
+"""Project controller module."""
+
+from tg import expose
+
+from acmwebsite.lib.base import BaseController
+from acmwebsite.model import DBSession, Project
+
+
+class ProjectsController(BaseController):
```

Comment:
Will there ever be the possibility of having a page for separate projects? If yes, the OO style thing we did with the meetings/surverys might be a good choice.

Not priority... something we _eventually_ might consider. Can be done way after this has been merged...

Reply:
Yes. That's why I had this as a separate controller. It's kinda dumb right now, but in the future, I can definitely see eventually having a page for each project (for example, it may include more details about the project and/or additional media such as screenshots). I think is should be pretty easy to add on a `ProjectController` for the individual projects with this setup.

Reply:
Individual projects should  be found at